game/core/src/me/lihq/game/screen/NavigationScreen.java

```diff
@@ -333,11 +333,21 @@ public void updateTiledMapRenderer()
         this.currentNPCS = game.getNPCS(game.player.getRoom());
     }
 
+    public boolean getChangeMap()
+    {
+        return(changeMap);
```

Comment:
The only way to test it would be to add a getter or make changeMap public. Without adding a couple getters, nothing in the NavigationScreen class can be tested :(